source/elements/oneDAL/source/data_management/accessors.rst

```diff
@@ -17,31 +20,21 @@ Requirements
 Each accessor implementation shall:
 
 1. Define a single :term:`format of the data <Data format>` for the
-   accessor. Every single accessor type shall return and use only one data
-   format.
-
-2. Provide an access to at least one in-memory :txtref:`dataset` implementation
-   (such as :code:`table` or its sub-types).
-
-3. Provide read-only, write-only, or read-write access to the data. If an
-   accessor supports several :txtref:`dataset` implementations to be passed in,
-   it is not necessary for an accessor to support all access modes for every
-   input object. For example, tables shall support a single read-only mode
-   according to their :txtref:`<table> concept` definition.
+   access. Every single accessor type shall return and use only one data format.
 
-4. Provide the names for read and write operations following the pattern:
+2. Provide read-only access to the data in the :txtref:`table` types.
 
-   - :code:`pull_*()` for reading
+3. Provide the :code:`pull()` operation for obtaining the values from the table.
 
-   - :code:`push_*()` for writing
-
-5. Be lightweight. Its constructors from :txtref:`dataset` implementations
-   shall not have heavy operations such as copy of data, reading, writing, any
-   sort of conversions. These operations shall be performed by heavy operations
-   :code:`pull_*()` and :code:`push_*()`. It is not necessary to have copy- or
+4. Be lightweight. Its constructors shall not have heavy operations such as copy
+   of data, reading, any sort of conversions. These operations shall be
+   performed by method :code:`pull()`. It is not necessary to have copy- or
    move- constructors for accessor implementations since it shall be designed
```

Comment:
Support of copy- and move- constructors by the accessor is not required since..

'it shall be designed for the use in a local scope' - what does it mean?

Reply:
Added an explanation